README.md

```diff
@@ -1 +1,5 @@
-# cloud-foundation-fabric
\ No newline at end of file
+# Cloud Foundation Toolkit - Fabric
+
+The Cloud Foundation Fabric provides end-to-end Terraform code examples for prototyping on GCP. Code from this repository can be used as the basis for starting to develop an infrastructure as code repository for GCP before adopting production-grade Cloud Foundation Toolkit modules.
```

Comment:
Nit: "can be used to lay automated infrastructure foundations to then deploy the product-specific Cloud Foundation Toolkit modules, or for rapid prototyping of networking and organizational design."

I would also add a link to an index of all CFT modules on "Cloud Foundation Toolkit" modules.